src/kzg10/mod.rs

```diff
@@ -100,17 +104,17 @@ impl<E: PairingEngine> KZG10<E> {
             );
 
             let affines = E::G2Projective::batch_normalization_into_affine(&neg_powers_of_h);
-            Some(affines.into_iter().map(|a| a.into()).collect())
+            Some(affines.into_iter().map(|a| a.prepare()).collect())
         } else {
             None
         };
 
         end_timer!(prepared_neg_powers_of_h_time);
 
-        let beta_h = h.mul(beta).into_affine();
+        let beta_h = h.mul(&beta).into_affine();
         let h = h.into_affine();
-        let prepared_h = h.into();
-        let prepared_beta_h = beta_h.into();
+        let prepared_h = h.prepare();
```

Comment:
Do you have a list of these compatibility changes somewhere? Can see it being useful that we have it published somewhere